selftests/deployment/roles/tests/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Avocado test 1/1 - Checking for the Avocado version
+  shell: "{{ temporary_dir.path | default('/usr') }}/bin/avocado --version"
+  changed_when: false
+
+- name: Avocado-VT test 1/5 - vt-bootstrap
+  shell: "{{ temporary_dir.path | default('/usr') }}/bin/avocado vt-bootstrap --yes-to-all --vt-skip-verify-download-assets"
+  when:
+    - ansible_facts['distribution'] != "Fedora"
```

Comment:
I don't think this should exclude Fedora, so I'll properly test them and let you know.  Same applies to other (x/5) tests.